recipe/meta.yaml

```diff
@@ -1,41 +1,39 @@
 {% set name = "backports-datetime-fromisoformat" %}
 {% set version = "2.0.0" %}
 
-
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/backports-datetime-fromisoformat-{{ version }}.tar.gz
```

Comment:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{name}}-{{ version }}.tar.gz
```

Reply:
@ggabernet this is nice to have but not mandatory. Also, we will probably archive this feedstock, so it is not worth working on it further. Thanks!